pinot-core/src/main/java/com/linkedin/pinot/core/io/util/PinotDataBitSet.java

```diff
@@ -39,6 +39,20 @@
     }
   }
 
+  public static int getNumBitsPerValue(int cardinality) {
```

Comment:
Can you put some comments on how we decide the number of bits per value in high level with some simple examples?

Reply:
Added